sources/interface/MyGLApplication.cpp

```diff
@@ -121,28 +121,29 @@ bool MyGLApplication::saveSceneFile()
 
 void MyGLApplication::addSound()
 {
-    QString qSoundFile = QFileDialog::getOpenFileName(
+    QStringList qSoundFiles = QFileDialog::getOpenFileNames(
         nullptr, _Q("", "Load sound"),
         QString(), _Q("", "Sound files (*.*)"));
 
-    if (qSoundFile.isEmpty())
-        return;
-    QDir qSoundDir = QFileInfo(qSoundFile).dir();
-
-    std::unique_lock<std::mutex> lock(::currentSceneMutex);
-    Scene *scene = ::currentScene;
-
-    AudioFile *af = scene->loadNewSample(qSoundFile.toStdString());
-    if (!af) {
-        // cannot load
+    if (qSoundFiles.isEmpty())
         return;
+    QString qSoundFile = "";
+    for (int i =0;i<qSoundFiles.count();i++) {
```

Comment:
Mettre le style de code en cohérence avec le reste (espacements).
`for (int i = 0; i < qSoundFiles.count(); i++) {`